CHANGES

```diff
@@ -1,3 +1,7 @@
+0.9.0
```

Comment:
Probably does not need a major version bump since the API is backward compatible..